client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -82,6 +82,17 @@ def test_direct_update(self, client, experiment_run, model_for_deployment):
         new_build_ids = get_build_ids(endpoint.get_status())
         assert len(new_build_ids) - len(new_build_ids.intersection(original_build_ids)) > 0
 
+    @pytest.mark.skip(reason="this test takes too long to complete")
```

Comment:
How long is "too long"? I'd expect this to take maybe 2 minutes, which would be acceptable for our nightly tests.

Reply:
100 seconds. I'll un-skip.

Reply:
Unskipped.

Reply:
Thanks! Good thinking though—definitely keep an eye out for those slow-running tests